lib/util/git/config.ts

```diff
@@ -22,5 +22,8 @@ export function simpleGitConfig(): Partial<SimpleGitOptions> {
       onClose: true,
       onExit: false,
     },
+    timeout: {
+      block: 10000,
```

Comment:
this should probably configurable via global config. 🤔

Reply:
@viceice @rarkins  Created a draft cause I need to make sure. Is there a default git config in lib/config ?

Reply:
Please create a new admin-only (global) config option gitTimeout an default it to 10000. That way we can experiment with increasing or decreasing it, if necessary.

Reply:
@rarkin shouldn't we create a global timeouts object instead, so we can add more configurable timeouts in future easily without migration?

Reply:
Should this be in the `RepoGlobalConfig` or `GlobalOnlyConfig` ?

Reply:
I think `GlobalOnlyConfig`

Reply:
It should be then initialized and fetched in `config/default.ts` and not `config/global.ts` right ?

Reply:
A bit conflicted cause in the relevant code sections for git such as git-submodules we are doing `GlobalConfig.get()` already.
Not sure if its the best way to also do `getConfig` from `config/defaults.ts` as well

Reply:
it should be initialized from `config/options/index.ts`, then it will be on global options and set to global config on repo init.

Checkout `GlobalConfig.set`

Reply:
@rarkins @viceice 
Shouldn't the `stage` of this type be repository if we want to access the value of it at the repo init ?
Overall I'm trying to access this config value at `simpleGitConfig()` so it can be returned there at every use because the `simple-git` configuration is defined there.